pkg/syncer/cnsoperator/controller/cnsnodevmattachment/cnsnodevmattachment_controller.go

```diff
@@ -416,27 +417,25 @@ func (r *ReconcileCnsNodeVMAttachment) Reconcile(ctx context.Context,
 					msg = fmt.Sprintf("failed to verify is VM CR is present with UUID: %s "+
 						"in namespace: %s", nodeUUID, request.Namespace)
 					recordEvent(ctx, r, instance, v1.EventTypeWarning, msg)
-					return reconcile.Result{RequeueAfter: timeout}, nil
+					return reconcile.Result{RequeueAfter: timeout}, "NotComputed", nil
 				}
 				if !isPresent {
 					msg = fmt.Sprintf("VM CR is not present with UUID: %s in namespace: %s. "+
 						"Removing finalizer on CnsNodeVMAttachment: %s instance.",
 						nodeUUID, request.Namespace, request.Name)
```

Comment:
Can we print a log message message after this indicating we're returning success when both VM and VM CR are not found?

Reply:
Makes sense. Added